governance.md

```diff
@@ -47,6 +47,12 @@ rule sets. Contributors with a record of consistent, responsible past
 contributions who are planning major contributions in the future could be
 considered to become qualified Maintainers.
 
+To be added as a maintainer to a repository in the bazelbuild organization,
+file an issue in the respective repository. After approval by the current
+maintainers, ask to be added to the [Bazel team](https://github.com/orgs/bazelbuild/people){: .external}
+by writing an e-mail to governance@bazel.build. After this step is completed,
```

Comment:
Perhaps the mail should tie back to the original issue, so that we know where to look for context. Otherwise, when we get mail to goverance@, we may have to hunt down why.